src/sdk/SpeakerIdentificationModel.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { Contracts } from "./Contracts";
+import {
+    VoiceProfile,
+    VoiceProfileType,
+} from "./Exports";
+
+export class SpeakerIdentificationModel {
```

Comment:
Needs doc tags

Reply:
Added for this and SpeakerVerificationModel